{{cookiecutter.project_dashed}}-src/setup.cfg

```diff
@@ -0,0 +1,4 @@
+[flake8]
+max-line-length=100
+exclude=.git,.hg,.tox,dist,doc,*egg,build
+ignore=E121,E123,E126,E226,E24,E704,W503,W504
```

Comment:
Can we add a comment describing the errors we are ignoring.  Also, does this list conform to the "blessed" ignore list if such a thing exists at Level 12?

Reply:
Done.